core/domain/state_domain.py

```diff
@@ -217,6 +220,11 @@ def validate(
                 % self.rule_specs)
 
         if self.tagged_skill_misconception_id is not None:
+            if validation_from_exploration:
+                raise utils.ValidationError(
```

Comment:
Please add this error to the "Raises" section in the docstring.

Reply:
Done